fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/NoAnalysis.kt

```diff
@@ -0,0 +1,242 @@
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+/**
+ * No analysis UI for review quality check content.
+ *
+ * @param onAnalyzeClick Invoked when the user clicks on the check review button.
+ * @param productRecommendationsEnabled The current state of the product recommendations toggle.
+ * @param onReviewGradeLearnMoreClick Invoked when the user clicks to learn more about review grades.
+ * @param onOptOutClick Invoked when the user opts out of the review quality check feature.
+ * @param onProductRecommendationsEnabledStateChange Invoked when the user changes the product
+ * recommendations toggle state.
+ * @param modifier Modifier to be applied to the composable.
+ */
+@Composable
+fun NoAnalysis(
+    onAnalyzeClick: () -> Unit,
+    productRecommendationsEnabled: Boolean,
+    onReviewGradeLearnMoreClick: (String) -> Unit,
+    onOptOutClick: () -> Unit,
+    onProductRecommendationsEnabledStateChange: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.spacedBy(16.dp),
+    ) {
+        ReviewQualityNoAnalysisCard(onAnalyzeClick)
+
+        ReviewQualityInfoCard(
+            onLearnMoreClick = onReviewGradeLearnMoreClick,
+        )
+
+        ReviewQualityCheckSettingsCard(
+            productRecommendationsEnabled = productRecommendationsEnabled,
+            onProductRecommendationsEnabledStateChange = onProductRecommendationsEnabledStateChange,
+            onTurnOffReviewQualityCheckClick = onOptOutClick,
+            modifier = Modifier.fillMaxWidth(),
+        )
+    }
+}
+
+@Composable
+private fun ReviewQualityNoAnalysisCard(
+    onAnalyzeClick: () -> Unit,
+) {
+    ReviewQualityCheckCard(
+        modifier = Modifier.fillMaxWidth(),
+    ) {
+        var isAnalyzing by remember { mutableStateOf(false) }
+
+        Image(
+            painter = painterResource(id = R.drawable.shopping_no_analysis),
+            contentDescription = null,
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(90.dp)
+                .padding(start = 6.dp, top = 8.dp, end = 9.dp, bottom = 10.dp),
```

Comment:
I'm not sure this is the best as it won't be centered then. I think it would be fine to have the same padding in all directions.